src/main/java/com/google/sps/data/Event.java

```diff
@@ -48,119 +46,54 @@
    * 
    * @param name name of the location (NOT address)
    * @param address exact address of the POI
-   * @param date date from user input travel day. Cannot be null.
-   * @param startTime start of activity (HHMM format)
+   * @param startTime start of activity (YYYY-MM-DD'T'HH:MM:SS)
    * @param travelTime time spent traveling to next location (minutes). 
    *        Null if last location of the day.
    * @param timeAtLocation time spent at POI (minutes)
    */
-  public Event(String name, String address, String date, int startTime, 
+  public Event(String name, String address, LocalDateTime startTime, 
               int travelTime, int timeAtLocation) {
     this.name = name;
     this.address = address;
-    this.date = date;
     this.startTime = startTime;
-    this.travelTime = travelTime;
-    this.endTime = calculateEndTime(timeAtLocation);
-  
-    createCalendarTimes();
+    this.endTime = startTime.plusMinutes(Long.valueOf(timeAtLocation));
+    this.travelTime = Long.valueOf(travelTime);
```

Comment:
why does travelTime need to be converted from an int to a long? If you're always going to get an 'int' why change it? It doesn't seem to be used as a long anywhere.

Reply:
In the LocalDateTime library the plusMinutes takes in a long not an int